core/src/main/scala/kafka/admin/ZkSecurityMigrator.scala

```diff
@@ -128,7 +129,7 @@ object ZkSecurityMigrator extends Logging {
     // Now override any set system properties with explicitly-provided values from the config file
     // Emit INFO logs due to camel-case property names encouraging mistakes -- help people see mistakes they make
     info(s"Found ${zkTlsConfigFileProps.size()} ZooKeeper client configuration properties in file $filename")
-    zkTlsConfigFileProps.asScala.foreach { case (key, value) =>
+    zkTlsConfigFileProps.asScala.forKeyValue { (key, value) =>
```

Comment:
Could we replace such ```asScala.forKeyValue``` by java ```Map.forEach``` ? We don't need to convert it to scala collection actually.

Reply:
See https://github.com/apache/kafka/commit/847ff8f55735568fe879d490af297e047b994f5e for an explanation why `asScala` still makes sense when dealing with `Properties`.